.gitignore

```diff
@@ -20,6 +20,4 @@ testpullfilecache*
 .DS_Store
 package-lock.json
 yarn.lock
-/.vs
-typings/types.d.ts
-typings/promiseBasedTypes.d.ts
```

Comment:
Annoyingly, TSLint was failing because these files weren't committed to the repo :upside_down_face:.